src/main/resources/blockly/js/mcreator_blocks.js

```diff
@@ -510,7 +510,7 @@ Blockly.Extensions.register('effect_list_provider',
 Blockly.Extensions.register('potion_list_provider',
     function () {
         this.appendDummyInput().appendField(new Blockly.FieldDropdown(
-            arrayToBlocklyDropDownArray(javabridge.getListOf("potion"))), 'potiontype');
+            arrayToBlocklyDropDownArray(javabridge.getListOf("potion"))), 'potionitem');

```

Comment:
The second name is the name used in procedure blocks when generating the code; if custom procedure blocks use this field, it will not work after this change.

Reply:
The data provider is being added with this PR, so there's no procedure block with this field yet

Reply:
Oops xD